### PR TITLE
Fservice show tickets and associated details

### PR DESCRIPTION
**Pull request by Chris-Blesson**
Hello @ravirajsubramanian I changed  jquery to vanilla js codes. Also I noticed a small issue , whenever I changed from the dropdown the navbar on the side was not updated, I aslo fixed that part.Kindly review my work Thanks in advance.

### CODE REVIEW

File: Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/modal.js [hunk 1]
```diff
@@ -1,97 +1,130 @@
...
+                    li.className = 'clearfix';
+                    li.appendChild(label);
+                    li.appendChild(value);
+                    targetContainer.appendChild(li);
                 }
+            }
+
+        })
+        .catch(function(e) {
+            console.log('Exception - ', e);
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/modal.js:66]: Can this be converted to console.error statement to differentiate from development time logs?
Also, please add a meaningful error message and log the error object in a separate log state to make sure there's no string conversion happens.

File: Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/modal.js [hunk 2]
```diff
@@ -1,97 +1,130 @@
...
+
 
-    $(document).ready(function() {
-        app.initialized().then(initModal);
-    });
-})(window.jQuery);
\ No newline at end of file
+(function() {
+    app.initialized().then(initModal);
+})();
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/modal.js:130]: Please add a new empty line at the end-of-file.

File: Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/template.html [hunk 3]
```diff
@@ -10,8 +9,9 @@
 <body>
     <div class="fw-widget-wrapper">
         <button class="btn" id="startDemo">Show Data</button>
+
     </div>
     <script src="app.js"></script>
 </body>
 
-</html>
+</html>
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/template.html:17]: Please add an empty line at EOF.

File: Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/app.js [hunk 4]
```diff
@@ -1,28 +1,27 @@
...
+            }
+        );
+};
+let initHandlers = function() {
+    document.getElementById("startDemo").addEventListener("click", initDemoApp);
+};
+
+(function() {
+    app.initialized().then(initAPP);
+})();
```
Comment (ravirajsubramanian) [inline Freshworks-Samples/Freshservice/show_ticket_and_associated_details/app/app.js:27]: Please add an empty line at the end-of-file.

**Comment by ravirajsubramanian** [comment]
@Chris-Blesson Great work at the refactoring. Please address the minor comments from the review.
Thank you!

**Comment by Chris-Blesson** [comment]
@ravirajsubramanian  Thank you for the review , I have addressed the changes. For new lines I am able to see them in my editor but while viewing in Github I am not sure whether it is reflected.

**Review by ravirajsubramanian — APPROVED** [review]

**Comment by ravirajsubramanian** [comment]
@Chris-Blesson Thanks for incorporating the suggested changes.

Yeah, the new line at the end of file will not be visible in the GitHub diff checker. Only when it's missing, there will be a warning sign.

Thanks for your contribution. 🙂

**Pull request merged**